### ISSUE TITLE
Nan when training optical flow channels with kinetics pretrained weights

### ISSUE THREAD

**Issue by mangdian** [yjxiong/action-detection#74]
When training the optical flow channel of SSN, the loss become nan after several iterations. Do you have any idea why this happens? Is there any difference between the training settings with imagenet pretrained weights and Kinetics pretrained weights?

**Comment by yjxiong** [comment]
I don't think there is any difference. Are you training BN-Inception or InceptionV3 CNN?

**Comment by mangdian** [comment]
@yjxiong I use BN-Inception. This problem only happens when I use BN-Inception network with optical flow as input

**Comment by yjxiong** [comment]
Are you exactly following the training parameters in the README?

**Comment by happygds** [comment]
@mangdian Hi, I met the same problem, did your solve it ?

**Comment by mangdian** [comment]
@happygds No, I have not solve this problem yet

**Comment by ShiinaMitsuki** [comment]
with BNInception. I solved this problem by decrease the initial learning rate.
lr=1e-3 --> nan
lr=1e-4 --> ok

**Comment by yjxiong** [comment]
See if #89 will solve your problem especially if you have fewer GPUs. It is merged to the master branch.
